src/http/article.controller.spec.js

```diff
@@ -64,9 +68,9 @@ describe("Article controller", () => {
                 expect(spies.create).toBeCalledWith(expect.any(CreateArticleInputDto));
             });
 
-            test("should respond with 200", async () => {
+            test("should redirect", async () => {
                 await controller.create(req, res);
-                expect(res.status).toBeCalledWith(200);
+                expect(res.redirect).toHaveBeenCalled();
```

Comment:
Optionally, we can also make sure it's redirecting to the specific route we expect

Reply:
It's been a while since I last touched the code (rip streak 🪦) but we're not receiving the id of the created article, maybe we need to look into the mock for that, I know there's a part of the code that we give a new id.
Could you take a look into that?

Reply:
We check for the documents saved during tests in ``ModelMock.collection``.
Just pushed an update ee12a6fe that adds the verifications I suggested.